Presentation.Web/app/components/global-admin/global-admin-misc.controller.ts

```diff
@@ -162,17 +160,17 @@
                 const nameAndEmail = `${$scope.selectedUser.text}, ${$scope.selectedUser.email}`;
 
                 if (confirm(`Er du sikker på, at du vil slette ${nameAndEmail}`)) {
-                    notify.addInfoMessage(`Sletter ${nameAndEmail}`);
+                    const message = notify.addInfoMessage(`Sletter ${nameAndEmail}`);
                     userService.deleteUser(id)
                         .then(() => {
-                                notify.addSuccessMessage(`Sletter ${nameAndEmail}`);
-                                $scope.selectedUser = null;
-                            }
+                            message.toSuccessMessage(`Sletter ${nameAndEmail}`);
+                            $scope.selectedUser = null;
+                            $state.reload();
```

Comment:
was not working because it was placed outside the promise context, so that the UI was refreshed before the promise was completed and the user was removed